src/types.ts

```diff
@@ -110,6 +110,29 @@ type JSONRPCResponseError = {
   id: string | number | null;
 };
 
+type ObjectEmpty = NonNullable<unknown>;
```

Comment:
We use this to mean `{}` in PK. Why is this written this way?

Reply:
i don't know, i'm assuming it's because the default type of `extends Record<string, JSONValue>` is `{}`

Reply:
sure